### PR TITLE
Revert back to old way of eval

### PR DESCRIPTION
**Pull request by hetunandu**
## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

**Review by mohanarpit — APPROVED** [review]

File: app/client/src/workers/evaluate.ts [hunk 1]
```diff
@@ -159,7 +175,7 @@ export default function evaluate(
       self[func] = undefined;
     });
     try {
-      result = Function(script)();
```
Comment (Rishabh-Rathod) [inline app/client/src/workers/evaluate.ts:162]: @hetunandu Could you help us understand why there was a need to revert this change? 

Also, did we miss the `new` keyword before the Function constructor, or was it intentional?

Comment (hetunandu) [reply app/client/src/workers/evaluate.ts:162]: This caused users existing anonymous functions to break. Hence we decided to revert it
